packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -82,17 +111,40 @@ enum PollState {
  * @type TokenRatesState
  *
  * Token rates controller state
- * @property contractExchangeRates - Hash of token contract addresses to exchange rates (single globally selected chain, will be deprecated soon)
- * @property contractExchangeRatesByChainId - Hash of token contract addresses to exchange rates keyed by chain ID and native currency (ticker)
+ * @property marketData - Hash of token contract addresses to market data
  */
 // This interface was created before this ESLint rule was added.
 // Convert to a `type` in a future major version.
 // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
 export interface TokenRatesState extends BaseState {
-  contractExchangeRates: ContractExchangeRates;
-  contractExchangeRatesByChainId: Record<
+  marketData: Record<
     Hex,
-    Record<string, ContractExchangeRates>
+    Record<
+      Hex,
+      {
+        tokenAddress: `0x${string}`;
+        value: number;
+        currency: string;
+        allTimeHigh: number;
+        allTimeLow: number;
+        circulatingSupply: number;
+        dilutedMarketCap: number;
+        high1d: number;
+        low1d: number;
+        marketCap: number;
+        marketCapPercentChange1d: number;
+        price: number;
+        priceChange1d: number;
+        pricePercentChange1d: number;
+        pricePercentChange1h: number;
+        pricePercentChange1y: number;
+        pricePercentChange7d: number;
+        pricePercentChange14d: number;
+        pricePercentChange30d: number;
+        pricePercentChange200d: number;
+        totalVolume: number;
+      }
```

Comment:
Is this different than the type on line 77? If yes, let's move to a single type to make future edits easier